mangum/adapter.py

```diff
@@ -111,9 +124,13 @@ def __call__(self, event: dict, context: dict) -> dict:
                 query_string = event.get("rawQueryString", "").encode()
             else:
                 source_ip = request_context.get("identity", {}).get("sourceIp")
-                multi_value_query_string_params = event[
-                    "multiValueQueryStringParameters"
-                ]
+                multi_value_query_string_params = {}
+                if event.get("multiValueQueryStringParameters"):
+                    use_multi_value_headers = True
+                    multi_value_query_string_params = event.get("multiValueQueryStringParameters")
+                elif event.get("queryStringParameters"):
+                    multi_value_query_string_params = {k:[v] for k, v in event.get("queryStringParameters").items()}
```

Comment:
If it isn't a lot of effort to include test coverage this, then please add it. Otherwise it's okay to add `pragma: no cover` for now.